engine/crates/federated-graph/src/render_sdl.rs

```diff
@@ -88,6 +88,8 @@ pub fn render_sdl(graph: &FederatedGraph) -> Result<String, fmt::Error> {
                 write_field(field.field_id, graph, &mut sdl)?;
             }
             writeln!(sdl, "}}\n")?;
+        } else {
+            sdl.push_str("\n\n");
```

Comment:
Empty types look weird otherwise. They should not happen, but it doesn't cost anything to render them properly anyway.